### PR TITLE
Changes for navbar for contact section

### PR DESCRIPTION
**Pull request by AL2002MI08**
We did the following :
Added the white shape for contact section 
Fixed the position on medium screens

### CODE REVIEW

**Review by tuyishimejohnson — CHANGES_REQUESTED** [review]
Can you make this div appear on one line?

File: src/sections/contact.html [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <section class="flex flex-col gap-36 pt-64">
-  <div
-    class="mx-auto flex-col px-3 text-customGreen lg:flex lg:w-8/12 lg:flex-row lg:gap-36"
-  >
+  <div class="mx-auto flex-col px-3 text-customGreen lg:flex lg:w-8/12 lg:flex-row lg:gap-36">
```
Comment (tuyishimejohnson) [inline src/sections/contact.html:2]: Can you change this div to appear on one line?

**Review by tuyishimejohnson — CHANGES_REQUESTED** [review]

File: src/sections/contact.html [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+    <img
```
[the remark this answers is not in the document]
Comment (AL2002MI08) [reply src/sections/contact.html:3]: Done. Thanks for the review

**Review by IvyMurage — APPROVED** [review]

**Review by tuyishimejohnson — APPROVED** [review]
The code looks good. Keep on!

**Pull request merged**